### PR TITLE
3_5B

### PR DESCRIPTION
**Pull request by rschlaikjer**
There's a bit of noise in this one because I moved all the `@g`s so they'd stop making my spellchecker mad, sorry

### CODE REVIEW

File: script/Arcueid/Day 3/03_00_ARC03_5B.txt [hunk 1]
```diff
@@ -325,78 +365,114 @@ All I want right now is to be left alone in my room. I want to take off my glass
...
+%{ri}
+           .seceip dnasuoht a otni reh raet ot tnaw I
+%{/ri}
 }
 [sha:1ad329634067cd29c3c285a6082fccc6a8d1f4d8]{
 -- Page 211, Offset 4530.
 -- @g　自慰や性交という代償ではなく、より単純で原始的な、
-@gNot as a replacement for masturbation or intercourse, but for a simpler, more primal desire.
+@g
+I'm not talking about masturbation or intercourse.
```
Comment (Avuxo) [inline script/Arcueid/Day 3/03_00_ARC03_5B.txt:426]: 代償 seems to be dropped here. Was that intentional?

File: script/Arcueid/Day 3/03_00_ARC03_5B.txt [hunk 2]
```diff
@@ -325,78 +365,114 @@ All I want right now is to be left alone in my room. I want to take off my glass
...
+%{ri}
+         .tnetnoc s'traeh ym ot reh ■ ot tiaw t'nac I
+%{/ri}
 }
 [sha:830c0b809564ada8a61f4919e3eb300f916d126b]{
 -- Page 211, Offset 4534.
 -- @g　その“初めて”を、覚えていない。
-@gI don't even remember the 'first time'.
+@g
+I don't even remember my 'first time'.
```
Comment (Avuxo) [inline script/Arcueid/Day 3/03_00_ARC03_5B.txt:457]: Is this line "my first sexual encounter", "my first time feeling this way", "the beginning of this state [as in the accident]", or "the start of these feelings"

File: script/Arcueid/Day 3/03_00_ARC03_5B.txt [hunk 3]
```diff
@@ -406,109 +482,129 @@ All I want right now is to be left alone in my room. I want to take off my glass
 [sha:e847e0eed66e419e1c97dfb250ed3867bc4c4812]{
 -- Page 213, Offset 4539. Mods: @n.
 -- @g　ノドがアツイ。
-// (candidate for font/italics change)
-@gMy THROAT is ACHING.
+@g
+My %{i}throat%{/i} is %{i}aching%{/i}.
```
Comment (Avuxo) [inline script/Arcueid/Day 3/03_00_ARC03_5B.txt:486]: I actually like the all caps on these lines. it reads real crazy-like

File: script/Arcueid/Day 3/03_00_ARC03_5B.txt [hunk 4]
```diff
@@ -518,87 +614,112 @@ All I want right now is to be left alone in my room. I want to take off my glass
...
-@gI press the call button. I go inside.
+@g
+I press the call button. I go inside.
 }
 [sha:e992a594172d20e17c2b39ad557dd30c02625520]{
 -- Page 219, Offset 4562.
 -- @g　途端、あまりの気配にぞくりときた。
-@gAt that moment, I'm overwhelmed by her remaining presence.
+@g
+I'm immediately overwhelmed by a sense of her presence.
```
Comment (Avuxo) [inline script/Arcueid/Day 3/03_00_ARC03_5B.txt:624]: merge these two vers imo. "I'm immediately overwhelmed by her remaining presence." Boy I wish lingering wasn't on the next line. "remaining scent" would sound terrible.

**Review by Avuxo — APPROVED** [review]
otsukare

File: script/Arcueid/Day 3/03_00_ARC03_5B.txt (hunk 1 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 3/03_00_ARC03_5B.txt:426]: My interpretation here was that this is that 自慰 and 性交 are methods to "pay down" (代償) his desire, in which case it felt a bit clunky to explicitly try and work that word in / hopefully clear from the context of the preceding connected line

Comment (Avuxo) [reply script/Arcueid/Day 3/03_00_ARC03_5B.txt:426]: I agree with that interpretation, but what I'm thinking is lost is that he's saying:
"this isn't an alternative to sexual behavior, it's an entirely different desire".  It's not just that he's not talking about masturbation or intercourse, it's that he's "not talking about" the need they would fulfill.

Maybe that distinction isn't actually necessary but that's at least what I was pointing out.

File: script/Arcueid/Day 3/03_00_ARC03_5B.txt (hunk 2 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 3/03_00_ARC03_5B.txt:457]: All the non-backwards lines are the same train of thought - 
> 最後に欲望を満たしたのは<何|い><歳|つ>だったか。
> 自慰や性交という代償ではなく、より単純で原始的な、
> 覚えていない。そもそもオレは、まだ、
> その“初めて”を、覚えていない。
> ...
> When was the last time I satisfied my desire?
> I'm not talking about masturbation or intercourse.
> I mean something simpler, more primal―――
> I don't remember what it is. In fact, I don't even―――
> I don't even remember my 'first time'.

My read here is that その初めて is when he first satisfied his desires, whenever that was (<何|い><歳|つ>だったか). 
If that's the right interpretation, we could make it more clear with e.g.
> I don't even remember the first time I satisfied it.

or, at risk of deviating too hard,
> I don't even remember ever having satisfied it at all.

File: script/Arcueid/Day 3/03_00_ARC03_5B.txt (hunk 4 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 3/03_00_ARC03_5B.txt:624]: Yeah I initially wrote lingering and then welp :aokoheh:
I don't love remaining either though, how about
> I'm immediately overwhelmed by the traces of her presence.

File: script/Arcueid/Day 3/03_00_ARC03_5B.txt (hunk 2 above)
Comment (Avuxo) [reply script/Arcueid/Day 3/03_00_ARC03_5B.txt:457]: ok, that's what I thought. The reason why I raised it is that "my first time" is very specific and feels like it runs counter to the line 2 lines above it. I like both of those suggestions but favor the latter.

File: script/Arcueid/Day 3/03_00_ARC03_5B.txt (hunk 4 above)
Comment (Avuxo) [reply script/Arcueid/Day 3/03_00_ARC03_5B.txt:624]: works for me

File: script/Arcueid/Day 3/03_00_ARC03_5B.txt (hunk 1 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 3/03_00_ARC03_5B.txt:426]: How about
> I'm not talking about something that can be satisfied by masturbation or intercourse. I mean something...

Comment (Avuxo) [reply script/Arcueid/Day 3/03_00_ARC03_5B.txt:426]: works for me

**Pull request merged**